docs/modules/hdfs/examples/getting_started/getting_started.sh.j2

```diff
@@ -36,7 +36,7 @@ echo "installing Operators with stackablectl"
 stackablectl operator install \
   commons={{ versions.commons }} \
   secret={{ versions.secret }} \
-  listener={{ versions.listener }} \
+  listener={{ versions.listener }} --listener-class-preset stable-nodes \
```

Comment:
I think for stackablectl we should use it's clever mechanism. E.g. on IONOS a LoadBalancer is better, as it keps running after a week and is stable.
The helm change LGTM :)

Reply:
Reverted to auto-presets: c8fa084

Reply:
Script successful with stackablectl:
```
...
Created file: testdata.txt with status {"FileStatuses":{"FileStatus":[
{"accessTime":1761812270594,"blockSize":134217728,"childrenNum":0,"fileId":16386,"group":"supergroup","length":597,"modificationTime":1761812271828,"owner":"stackable","pathSuffix":"testdata.txt","permission":"644","replication":1,"storagePolicy":0,"type":"FILE"}
]}}
Delete file
File was deleted!
```

Reply:
Awesome, thanks!